projects/plugins/jetpack/extensions/shared/components/block-nudge/index.jsx

```diff
@@ -45,7 +47,17 @@ export default function BlockNudge( {
 				{ icon }
 				<span className="jetpack-block-nudge__text-container">
 					<span className="jetpack-block-nudge__title">{ title }</span>
-					{ subtitle && <span className="jetpack-block-nudge__message">{ subtitle }</span> }
+					{ ( subtitle || readMoreUrl ) && (
```

Comment:
I took into account that either subtitle or `readMoreUrl` might be missing, but I'll just gate with subtitle alone.